### DISCUSSION TITLE
How can see/load remoteEntry.js when deveveloping

### DISCUSSION THREAD

**Discussion by quangnv13** [originjs/origin.js#173] [Q&A]
I use federation in my application. It's work perfectly but i need run `vite build && vite preview` to create **remoteEntry.js** file. I would like to see/load **remoteEntry.js** file when developing with `npm run vite` like webpack with module federation.
![image](https://user-images.githubusercontent.com/64973827/211498734-6dc79978-68a7-4c81-8ac0-614d65682c7a.png)

**Comment by flyfishzy** [comment]
Please use `vite build --watch` to enable watch mode. It would be better to discuss vite-plugin-federation here https://github.com/originjs/vite-plugin-federation/discussions .

**Reply by Seifenn** [reply]
You can try this method 

-  Build your application using npm run build

-  Then install http-server with npm install http-server
- Launch the server with http-server [Pathtoyourdistdirectory] --port 4178 --cors
- and then you can access the localhost:4178/assets/remoteEntry.js
